appveyor.yml

```diff
@@ -0,0 +1,45 @@
+# Copyright (c) 2014, Ruslan Baratov

```

Comment:
Not sure how to handle license. Took this from my CMake mentor @ruslo's repo here: https://github.com/forexample/appveyor-test/blob/master/appveyor.yml.